modules/core/package.json

```diff
@@ -69,7 +69,7 @@
     "@turf/union": ">=4.0.0",
     "@turf/transform-translate": ">=4.0.0",
     "cubic-hermite-spline": "^1.0.1",
-    "deck.gl": ">=6.0.5",
+    "deck.gl": "6.3.2",
```

Comment:
This will create issues in the monorepo. At least relax to all compatible patch versions (~6.3.2)?